build_scripts/vcpkg.json

```diff
@@ -3,7 +3,7 @@
     "dependencies": [
         "ceres",
         "nlohmann-json",
-        "openimageio",
+        { "name": "openimageio", "features": [ "libraw" ] },
```

Comment:
In previous windows build dependency was brought in as:
```
openimageio:x64-windows@3.0.9.1#1
```
with this change it is now
```
openimageio[core,libraw]:x64-windows@3.0.9.1#1
```